wagtailio/project_styleguide/templates/patterns/includes/svg_sprites.html

```diff
@@ -119,6 +119,10 @@
         <path d="M74.406 78.238l-32.835-4.606-8.919-63.458 32.836 4.606 8.918 63.458zm-3.703-64.526l19.7-11.32c2.327-1.332 5.54.06 5.933 2.855l7.741 55.074c.231 1.644-.548 3.262-2.122 3.987L79.621 77.17 70.703 13.71zM4.94 24.127l22.498-12.885L36.356 74.7 16.491 86.042c-2.327 1.333-5.376-.082-5.769-2.877L2.982 28.09c-.23-1.644.383-3.24 1.958-3.964z" fill="currentColor"/>
     </symbol>
 
+    <symbol id="magnifying-glass" viewBox="0 0 21 22">
```

Comment:
Suggestion: this [icon exists here](https://github.com/wagtail/wagtail.org/blob/next/wagtailio/project_styleguide/templates/patterns/includes/svg_sprites.html#L114) however the one you've added works better with the `icon.html` template so let's remove the one i've linked to and keep yours.